stable/rabbitmq/values.yaml

```diff
@@ -139,7 +139,7 @@ securityContext:
 
 persistence:
   ## this enables PVC templates that will create one per pod
-  enabled: false
+  enabled: true
```

Comment:
do we want to enable persistence by default now? is it standard in other `stable/` charts?
if so, we should at least update the default value in `README.md` too.

Reply:
I don't know why it was disabled before but yes, I think we should enable it. 
I will update the value in the README

Reply:
OK.
I did a quick `grep` and a large majority of charts enable persistence by default.